interop/src/test/java/net/corda/v5/interop/HoldingIdAliasGroupInfoTest.java

```diff
@@ -0,0 +1,40 @@
+package net.corda.v5.interop;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class HoldingIdAliasGroupInfoTest {
+
+    private final HoldingIdAliasGroupInfo holdingIdAliasGroupInfo = mock(HoldingIdAliasGroupInfo.class);
+
+    @Test
+    void getShortHash() {
```

Comment:
Those tests test Mockito framework, the don't test the code business logic, please remove the whole class.